src/explore-education-statistics-common/src/modules/charts/components/MapGeoJSON.tsx

```diff
@@ -107,6 +106,8 @@ export default function MapGeoJSON({
     }
   }, [map, selectedFeature]);
 
+  const resetZoom = () => map.setZoom(5);
```

Comment:
Do we need to reset the position also? the default position is available in the parent component so could be used if so